packages/cli/src/zosfiles/edit/Edit.options.ts

```diff
@@ -0,0 +1,43 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandOptionDefinition, IO } from "@zowe/imperative";
+
+import i18nTypings from "../-strings-/en";
+
+// Does not use the import in anticipation of some internationalization work to be done later.
+const strings = (require("../-strings-/en").default as typeof i18nTypings).EDIT.OPTIONS;
+const defaultEditor = IO.getDefaultTextEditor();
+/**
+ * Object containing all options to be used by the View API
+ */
+export const EditOptions: { [key: string]: ICommandOptionDefinition } = {
+
+    /**
+     * The option to set a default editor
+     * @type {ICommandOptionDefinition}
+     */
+    editor: {
+        name: "editor",
+        aliases: ["ed"],
+        description: strings.EDITOR,
+        defaultValue: process.env['ZOWE_EDITOR'] ?? defaultEditor,
```

Comment:
I think we were considering to deprecate `ZOWE_EDITOR` in other commands that currently use it.
Do we need to support `ZOWE_EDITOR` for this command?
Or is it sufficient that `ZOWE_OPT_EDITOR` is already supported?